lib/instrumentation/span.js

```diff
@@ -96,6 +97,10 @@ Span.prototype.setDestinationContext = function (context) {
   this._destination = Object.assign(this._destination || {}, context)
 }
 
+Span.prototype.setMessagingContext = function (context) {
```

Comment:
In the function and var names you used "messaging" rather than the spec'd field name "message". Is there a reason for not having those be a match, to possibly assist with grepping?

The others (setDbContext, setHttpContext, setDestinationContext) all match the payload field name.

Reply:
Hmm, that's an interesting point.  This wasn't a conscious choice but (guessing) I'd say it fell our of the span's type being specced as `messaging` and not `message`.  This feels like a six of one situation (there's grep considerations on both side with any [gerund](https://www.dictionary.com/browse/gerund)), but it's a quick change.  Done: https://github.com/elastic/apm-agent-nodejs/commit/6c88f7fa64babaa69abb3ca9898f734f9cb6e6e5